apps/condo/pages/settings/hint/create.tsx

```diff
@@ -37,6 +37,6 @@ const CreateTicketPropertyHintPage = () => {
     )
 }
 
-CreateTicketPropertyHintPage.requiredAccess = OrganizationRequired
+CreateTicketPropertyHintPage.requiredAccess = SettingsReadPermissionRequired
```

Comment:
Should there be a `canManageTicketPropertyHints` check here?

And should there be similar checks on other settings pages if they are needed to manage objects?

Reply:
When we add (and if add) these permissions to the settings table, then we update the access to the pages.